docs/data-sources/certificate.md

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Compute Engine"
+layout: "ionoscloud"
+page_title: "IonosCloud: certificate"
+sidebar_current: "docs-datasource-certificate"
+description: |-
+  Get Information on a certificate
+---
+
+# ionoscloud\_ionoscloud_certificate
+
+The **Certificate data source** can be used to search for and return an existing certificate.
+You can provide a string for either id or name parameters which will be compared with provisioned certificates.
+If a single match is found, it will be returned. If your search results in multiple matches, an error will be returned.
+When this happens, please refine your search string so that it is specific enough to return only one result.
+
+## Example Usage
+
+### By ID
+```hcl
+data "ionoscloud_certificate" "example" {
+  id			= <certificate_id>
+}
+```
+
+### By Name
```

Comment:
Is it possible to also add partial match?

Reply:
yes, but in this case i guess we can have in if an or for partial match and substring match instead of equal